src/components/HTMLEditor.js

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { upload } from '../lib/api';
-import stylesheet from '../../node_modules/react-quill/dist/quill.snow.css';
 import classNames from 'classnames';
 
+import { upload } from '../lib/api';
+
+import '../../node_modules/react-quill/dist/quill.snow.css';
```

Comment:
what's the difference?

Reply:
Ideally, we should discuss that in the relevant PR:
https://github.com/opencollective/opencollective-frontend/pull/850

This will use the `next-css` plugin and all styles will be made available in a single stylesheet compiled by webpack without having to use the `<style dangerouslySetInnerHTML={{ __html: stylesheet }} />` trick.

Reply:
ok thanks for the info.